src/Host.h

```diff
@@ -161,6 +161,7 @@ class Host  : public QObject
     bool               mAllowToSendCommand;
     bool               mAutoClearCommandLineAfterSend;
     bool               mBlockScriptCompile;
+    bool               mEchoLuaErrors; 
```

Comment:
Saw this has been done now in https://github.com/Mudlet/Mudlet/pull/349/commits/b32a56788c364f0683a1724caac1a20e4cbeb7a9. :+1: